app/assets/stylesheets/common.scss

```diff
@@ -581,6 +581,15 @@ header .search_forms,
   }
 }
 
+.search_forms {
+  .geolocate {
+    transition: filter 0.15s ease-in-out;
```

Comment:
We don't use transitions on any other button content, so I don't think it's necessary here